### DISCUSSION TITLE
如何获取好友的分组信息呢

### DISCUSSION THREAD

**Discussion by ellieot** [takayama-lily/oicq#187] [Q&A] [answered]
比如同事、同学等分组信息

**Comment by takayama-lily** [comment]
尚未支持

**Comment by Cloud370** [comment, marked as answer]
你可以带上qq空间的cookie 请求这个API
`https://mobile.qzone.qq.com/friend/mfriend_list?qzonetoken=&g_tk=计算之后的g_tk&res_uin=你的QQ&res_type=normal&format=json&count_per_page=10&page_index=0&page_type=0&mayknowuin=&qqmailstat=`

**Reply by clansty** [reply]
好诶
